src/StreamJsonRpc/JsonRpc.cs

```diff
@@ -230,10 +230,26 @@ public static JsonRpc Attach(Stream sendingStream, Stream receivingStream, objec
         }
 
         /// <summary>
+        /// Adds the specified target as possible object to invoke when incoming messages are received.  The target object
+        /// should not inherit from each other and are invoked in the order which they are added.
+        /// </summary>
+        /// <param name="target">Target to invoke when incoming messages are received.</param>
+        /// <remarks>This method must be called before JsonRpc starts listening for messages.</remarks>
+        public void AddLocalRpcTarget(object target)
+        {
+            Requires.NotNull(target, nameof(target));
+            Verify.Operation(!this.startedListening, Resources.BothReadableWritableAreNull);
```

Comment:
That's so weird.  I swear when I wrote this it was the right message.  I'll change it.